Scripts/Runtime/Entities/TaskDriver/TaskDriverManagementSystem.cs

```diff
@@ -166,6 +170,29 @@ public void RegisterTaskDriver(AbstractTaskDriver taskDriver)
             m_AllTaskDriverSystems.Add(taskDriver.TaskDriverSystem);
         }
 
+        public AccessController GetOrCreateCDFEAccessController<T>()
+            where T : struct, IComponentData
+        {
+            return GetOrCreateUnityEntityDataAccessController(typeof(T));
+        }
+
+        public AccessController GetOrCreateDBFEAccessController<T>()
+            where T : struct, IBufferElementData
+        {
+            return GetOrCreateUnityEntityDataAccessController(typeof(T));
+        }
+        
+        private AccessController GetOrCreateUnityEntityDataAccessController(Type type)
+        {
+            if (!m_UnityEntityDataAccessControllers.TryGetValue(type, out AccessController accessController))
```

Comment:
We do this so often I'm starting to think we should just create a `LazyDictionary` that auto-instantiates values for a requested key.

Reply:
It's not a bad idea, it just gets hard when the element doesn't have a parameterless constructor because then you can't generically create it.